src/main/java/com/wolox/training/repositories/UsersRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.models.Users;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UsersRepository extends JpaRepository<Users, Long> {
+
+    public Optional<Users> findByUserName(String userName);
```

Comment:
It's no necessary add public to method in the repositories.

Reply:
Yes, thanks! I pushed the change